From: Ben Hutchings <ben@decadent.org.uk>
Subject: inet_frag: Avoid ABI change in 3.2.42
Date: Mon, 01 Apr 2013 00:45:25 +0100
Forwarded: not-needed

Hide an added #include from genksyms, as it will otherwise change
symbol versions.
---
--- a/net/ipv4/inet_fragment.c
+++ b/net/ipv4/inet_fragment.c
@@ -21,7 +21,9 @@
 #include <linux/rtnetlink.h>
 #include <linux/slab.h>
 
+#ifndef __GENKSYMS__
 #include <net/sock.h>
+#endif
 #include <net/inet_frag.h>
 
 static void inet_frag_secret_rebuild(unsigned long dummy)
